File: onebot-sdk/src/main/java/cn/evole/onebot/sdk/response/group/GroupInfoResp.java
package cn.evole.onebot.sdk.response.group;

import com.google.gson.annotations.SerializedName;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.EqualsAndHashCode;
import lombok.NoArgsConstructor;

/**
 * Created on 2022/7/8.
 *
 * @author cnlimiter
 */
@Data
@AllArgsConstructor
@NoArgsConstructor
@EqualsAndHashCode(callSuper = true)
public class GroupInfoResp extends GroupDataResp{

    @SerializedName("group_memo")
    private String groupMemo;

    @SerializedName("group_create_time")
    private int groupCreateTime;

    @SerializedName("group_level")
    private int groupLevel;

    @SerializedName("member_count")
    private Integer memberCount;

    @SerializedName("max_member_count")
    private Integer maxMemberCount;

}

File: onebot/src/main/kotlin/sdk/response/group/GroupInfoResp.kt
package cn.evolvefield.onebot.sdk.response.group

import com.google.gson.annotations.SerializedName
import lombok.AllArgsConstructor
import lombok.Data
import lombok.EqualsAndHashCode
import lombok.NoArgsConstructor

@Data
@AllArgsConstructor
@NoArgsConstructor
@EqualsAndHashCode(callSuper = true)
class GroupInfoResp : GroupDataResp() {
    @SerializedName("group_memo")
    var groupMemo = ""
    @SerializedName("group_create_time")
    var groupCreateTime = 0
    @SerializedName("group_level")
    var groupLevel = 0
    @SerializedName("member_count")
    var memberCount: Int? = null
    @SerializedName("max_member_count")
    var maxMemberCount: Int? = null
}
